lib/widgets/event_item.dart

```diff
@@ -374,9 +383,9 @@ class EventItem extends StatelessWidget {
         return _buildItem(
           context: context,
           spans: [
-            TextSpan(text: ' forked '),
+            TextSpan(text: ' ${AppLocalizations.of(context)!.forked} '),
             _buildRepo(context, '$forkeeOwner/$forkeeName'),
-            TextSpan(text: ' from '),
+            TextSpan(text: ' ${AppLocalizations.of(context)!.from} '),
             _buildRepo(context),
           ],
```

Comment:
{somebody} forked {repo1} from {repo2}, the order of these parts may be different in other languages.

I guess the best way is like function parameters, for example, `Translate('somebody', 'repo1', 'repo2')`, which is more flexible.

But I'm not sure if Flutter/WebLate already supports this way.

Reply:
Ah I see. There seems to be a way to handle this case as well as plurals in the dart intl package. I'll take a look

Reply:
@pd4d10 if the entire text is a part of TextSpan() there seems to be no problem. But if there's a buildRepo or some other widget that uses the text I'm not sure of how to do it. For example in the line you've marked above, I could pass translate(somebody, repo1, repo2) and then in the translated version I could find and replace the repo1 and repo2 parts with buildRepo. Not sure if its the best way to do it

Reply:
Oh, you are right. It seems not possible if it's built with Widget or TextSpan instead of pure text.

In the pure text case, [intl](https://github.com/dart-lang/intl#messages) seems to support parameters. For example: 

English translation: `{somebody} forked {repo1} from {repo2}`
Chinese translation: `{somebody} from {repo2} forked {repo1}`

But if `{somebody}` or `{repo}` is wrapped with a widget, there is no way to do that.

Reply:
I guess it's ready to merge if we can't figure out a better way. Could you please solve the conflicts?

Reply:
> In the pure text case, intl seems to support parameters

Yeah I was doing this. I'm still changing all the text that uses only TextSpan. I'll do that and handle plurals.